components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -247,35 +246,53 @@ public DataSection phenotypeBody(Patient patient) throws Exception
         if (sortedFeatures.size() == 0) {
             // offset is included to account for the presence of both "positive" and "negative" in "present"
             int offset = bothTypes ? 1 : 0;
-            Integer emptyX = 0;
             for (int i = 0; i < present.size() - offset; i++) {
-                DataCell cell = new DataCell("", emptyX, 0);
+                DataCell cell = new DataCell("", i, 0);
                 section.addCell(cell);
-                emptyX++;
             }
         }
         // section.finalizeToMatrix();
         return section;
     }
 
+    public void variantsSetup(Set<String> enabledFields) throws Exception
+    {
```

Comment:
My thoughts are as follows: the translation ability of export is not so urgent to be made, thus, it can wait to be done all at once, till we merge variants, which will happen soon. 

Orion Buske <notifications@github.com> wrote:

>>              }
>>          }
>>          // section.finalizeToMatrix();
>>          return section;
>>      }
>>  
>> +    public void variantsSetup(Set<String> enabledFields) throws Exception
>> +    {
>
>Either variants should be removed from this or I don't think this PR should have been made until Variants was merged and you had rebased. A large fraction of the code people are reviewing here is irrelevant to the PR.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/phenotips/phenotips/pull/1905/files#r43185402